lua/lspconfig/configs.lua

```diff
@@ -172,6 +173,12 @@ function configs.__newindex(t, config_name, config_def)
       end)
     end
 
+    function M.shutdown()
```

Comment:
why does it use a randomly different term instead of "stop"?
```suggestion
    function M.stop()
```